backend/src/main/java/com/yigongil/backend/application/StudyService.java

```diff
@@ -114,6 +133,48 @@ public void apply(Member member, Long studyId) {
         );
     }
 
+    @Transactional
+    public void createRegularFeedPost(Member member, Long studyId, RegularFeedPostCreateRequest request) {
+        RegularFeedPost regularFeedPost = RegularFeedPost.builder()
+                                                         .author(member)
+                                                         .study(findStudyById(studyId))
+                                                         .imageUrl(request.imageUrl())
+                                                         .content(request.content())
+                                                         .build();
+        feedPostRepository.save(regularFeedPost);
+    }
+
+    @Transactional
+    public void createCertificationFeedPost(Member member, Long id, CertificationFeedPostCreateRequest request) {
+        Study study = findStudyById(id);
+        CertificationFeedPost feedPost = CertificationFeedPost.builder()
+                                                              .author(member)
+                                                              .study(study)
```

Comment:
p4: 별 문제는 아니지만, 코드 일관성을 위해 createRegularFeedPost()와 비슷한 형태로 수정하면 어떨까요??

```suggestion
    public void createCertificationFeedPost(Member member, Long studyId, CertificationFeedPostCreateRequest request) {
        CertificationFeedPost feedPost = CertificationFeedPost.builder()
                                                              .author(member)
                                                              .study(findStudyById(studyId))
```